Loop/Managers/DeviceDataManager.swift

```diff
@@ -366,6 +375,14 @@ final class DeviceDataManager: CarbStoreDelegate, CarbStoreSyncDelegate, DoseSto
 
         device.assertIdleListening()
 
+        assertCurrentPumpStatus(device: device) {
+            if UserDefaults.standard.fetchPumpGlucoseEnabled {
+                self.assertCurrentEnliteData(device: device)
```

Comment:
This needs to be synchronized with fetching pump data.  As it stands, it looks like pump data and enlite data could both get onto their own independent 5 min cycles.